### PR TITLE
Fix default value of cluster revision attribute on identify cluster

### PR DESCRIPTION
**Pull request by PSONALl**
#### Problem
TE 8 Identify Cluster - Read Cluster Revision global attribute return version 2 instead of 4
* Test case: [TC-I-1.1 Global attributes with server as DUT](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/identify.adoc#311-tc-i-11-global-attributes-with-server-as-dut) -> [step2](https://github.com/CHIP-Specifications/chip-test-plans/blob/master/src/cluster/identify.adoc#:~:text=Verify%20that%20the%20DUT%20response%20indicates%20that%20the%20ClusterRevision%20attribute%20has%20the%20value%204)
* The cluster revision attribute default value must be 4 according to chip-test-plans
* if cluster revision value = 4 it indicates revision has new data model format and notation according to Matter Specs

#### Change overview
* changed default value in all-clusters-app.zap file
* regenerated zap

#### Testing
How was this tested? 
* Tested commissioning and cluster control using chip-tool

### CODE REVIEW

**Review by bzbarsky-apple — APPROVED** [review]
Seems fine, but do we need to fix it for other apps too?

**Review by andy31415 — APPROVED** [review]

**Comment by andy31415** [comment]
ZAP failures seems to indicate that restyle-diff was maybe not run.

**Review by dhrishi — APPROVED** [review]

**Comment by PSONALl** [comment]
> Seems fine, but do we need to fix it for other apps too?

Yes, need to add the same change for other apps too

**Comment by PSONALl** [comment]
> ZAP failures seems to indicate that restyle-diff was maybe not run.

I will do the same change in other apps and then will do restyle

**Comment by markus-becker-tridonic-com** [comment]
Similar change would be required for On/Off cluster as well: https://github.com/CHIP-Specifications/chip-test-plans/issues/282.

**Comment by PSONALl** [comment]
> Seems fine, but do we need to fix it for other apps too?

@bzbarsky-apple should this change be done for all the clusters too?

**Comment by bzbarsky-apple** [comment]
> @bzbarsky-apple should this change be done for all the other clusters too?

Maybe, but those should be separate PRs.  One PR per cluster that needs this.

**Comment by PSONALl** [comment]
@bzbarsky-apple @andy31415 @dhrishi Added default value of cluster revision =4 and also max allowed value of attribute IdentifyTime is changed to 65534 (0xFFFE) in xml and regenerated all the zap files for all the applications

File: src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml [hunk 1]
```diff
@@ -51,7 +51,7 @@ limitations under the License.
     <client tick="false" init="false">true</client>
     <server tick="false" init="false">true</server>
     <globalAttribute side="either" code="0xFFFD" value="2"/>
-    <attribute side="server" code="0x0000" define="IDENTIFY_TIME" type="INT16U" min="0x00" max="0xFE" writable="true"  default="0x0" optional="false">identify time</attribute>
+    <attribute side="server" code="0x0000" define="IDENTIFY_TIME" type="INT16U" min="0x00" max="0xFFFE" writable="true"  default="0x0" optional="false">identify time</attribute>
```
Comment (bzbarsky-apple) [inline src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml:54]: The right max per the spec is 0XFFFF, so we should actually just remove the min/max bits here altogether:
```suggestion
    <attribute side="server" code="0x0000" define="IDENTIFY_TIME" type="INT16U" writable="true"  default="0x0" optional="false">identify time</attribute>
```

**Review by bzbarsky-apple — APPROVED** [review]

File: src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml (hunk 1 above)
Comment (PSONALl) [reply src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml]: Added this change and regenerated zap

**Comment by dhrishi** [comment]
@PSONALl Please resolve the conflicts

**Comment by PSONALl** [comment]
> @PSONALl Please resolve the conflicts

done

**Review by msandstedt — APPROVED** [review]

**Pull request merged**